Cargo.toml

```diff
@@ -116,7 +116,13 @@ pretty_assertions = "=1.3.0"
 rand = "=0.8.5"
 regex = "^1.7.0"
 lazy-regex = "2.5.0"
-reqwest = { version = "0.11.18", default-features = false, features = ["rustls-tls", "stream", "gzip", "brotli", "socks"] }
+reqwest = { version = "0.11.18", default-features = false, features = [
+  "rustls-tls",
+  "stream",
+  "gzip",
+  "brotli",
+  "socks",
```

Comment:
What did you use to format these `Cargo.toml` files?

Reply:
I think that's "Even Better TOML's fault" I intended to roll this back once the deps stopped changing